### DISCUSSION TITLE
Anaconda : Python新手大禮包

### DISCUSSION THREAD

**Discussion by lx3211** [1101-GS4719-C/Python-Community-GS4719-C#380] [General]
Anaconda可以說是python的懶人包，裡頭除了python本身之外（python2,3），還包含了機器學習、資料分析、視覺化的套件。
ex. Numpy, Pandas, Seaborn...

**優點：** 省時，一次裝完新手會用到的幾乎所有的套件，剩下再用pip install的指令再去載就可以了

**缺點：** 很肥很肥，用不到的套件就是佔空間

[新手大禮包領取處(anaconda安裝教學)](https://ithelp.ithome.com.tw/articles/10192460?sc=hot)

**Comment by pekoranoinu** [comment]
感謝大佬分享，讓我這個新手一次全部下載，不用再花時間一個一個慢慢下。

**Comment by sean1325418** [comment]
Anaconda是真的很方便，下載套件什麼的很直觀，很推比較不想慢慢一個一個下載的人可以使用
